logstash-core/src/test/java/org/logstash/ackedqueue/QueueTest.java

```diff
@@ -755,4 +717,51 @@ public void getsPersistedByteSizeCorrectlyForUnopened() throws Exception {
             assertThat(q.getPersistedByteSize(), is(0L));
         }
     }
+    
+    private void stableUnderStress(final int capacity) throws IOException {
+        Settings settings = TestSettings.persistedQueueSettings(capacity, dataPath);
+        final ExecutorService exec = Executors.newScheduledThreadPool(2);
+        try (Queue queue = new Queue(settings)) {
+            final int count = 20_000;
+            final int concurrent = 2;
```

Comment:
@ph not so sure about that, for now we're just using one set of values here, no need to duplicate those upstream right?